Objective-C/CBLFullTextIndexConfiguration.h

```diff
@@ -25,7 +25,7 @@ NS_ASSUME_NONNULL_BEGIN
 @interface CBLFullTextIndexConfiguration : CBLIndexConfiguration
 
 /**
- Set the true value to ignore accents/diacritical marks. The default value is false.
+ Set the true value to ignore accents/diacritical marks. 
```

Comment:
Better to link to constant variable.

https://developer.apple.com/documentation/xcode/formatting-your-documentation-content#Link-to-symbols-and-other-content

Reply:
For objc, in this case, this value is not nullable. So user has to provide either NO or YES. So there is no default value in this case.